### PR TITLE
Google SafeBrowsing API

### PR DESCRIPTION
**Pull request by sheldonkreger**

### CODE REVIEW

File: pivoteer/models.py [hunk 1]
```diff
@@ -144,6 +144,18 @@ def historical_whois(self, indicator):
 
         return unique_records
 
+    def safebrowsing_record(self, indicator):
+        record_type = 'SB'
+        # time_frame = datetime.datetime.utcnow() + datetime.timedelta(hours=-24)
+        if check_domain_valid(indicator):
+            records = self.get_queryset().filter(Q(record_type=record_type),
+                                                 # Q(info_date__lt=time_frame),
+                                                 Q(info__contains=indicator))
```
Comment (bauerm) [inline pivoteer/models.py]: Contains might be too vague and return more than the relevant results ... "evil.somewhere.com" contains "somewhere.com", for example.

File: templates/pivoteer/pivoteer.html [hunk 2]
```diff
@@ -67,6 +67,13 @@
             </a>
         </li>
 
+        <li role="presentation">
+           <a href="#SafeBrowsingPanel" aria-controls="SafeBrowsingPanel" role-"tab data-toggle="tab">
+            SafeBrowsing
+            <span class = "glyphicon" aria-hidden=""true"</span>
```
Comment (sbgoodm) [inline templates/pivoteer/pivoteer.html]: this html is malformed. Missing closing `>` on the span. Also, do you want a glyphicon displaying here?

**Comment by bauerm** [comment]
+1

**Comment by sbgoodm** [comment]
+1

**Pull request merged**